tests/Config/ConfigCreatorFacadeTest.php

```diff
@@ -55,7 +55,7 @@ final class ConfigCreatorFacadeTest extends TestCase
     private $creatorFacade;
 
     /**
-     * @var LocatorInterface|MockObject
+     * @var \Infection\Locator\Locator|MockObject
```

Comment:
Just `Locator|MockObject` ?

Reply:
Should actually be `&`, but should be imported yes